static/js/containers/LearnerPage_test.js

```diff
@@ -59,9 +59,14 @@ import {
   clearSelectField,
   GoogleMapsStub,
 } from '../util/test_utils';
+import StaffLearnerInfoCard from '../components/StaffLearnerInfoCard';
+import {
+  REQUEST_DASHBOARD,
+  RECEIVE_DASHBOARD_SUCCESS,
+} from '../actions/dashboard';
 
 describe("LearnerPage", function() {
-  this.timeout(10000);
+  this.timeout(15000);
```

Comment:
Actually, looks like the answer is in Travis - tests failed because even 15s wasn't enough.